legion/legion/templating.py

```diff
@@ -0,0 +1,170 @@
+#
```

Comment:
templating.py, TemplateSystem, ltemplate

We need to use the same name everywhere. `template`?